src/components/Button/index.tsx

```diff
@@ -16,27 +17,33 @@ type ButtonVariant = (typeof BUTTON_VARIANTS)[keyof typeof BUTTON_VARIANTS];
 interface ButtonProps extends MantineButtonProps {
   buttonType: ButtonVariant;
   onClick?: () => void;
-  children: React.ReactNode;
+  linkTo?: string;
+  name: string;
 }
 
 export const Button: React.FC<ButtonProps> = ({
   buttonType,
-  children,
+  linkTo,
+  name,
   ...rest
 }) => {
   const { classes } = useStyles();
-  return (
-    <MantineButton
-      className={
-        buttonType === "primary"
-          ? classes.primary
-          : buttonType === "secondary"
-          ? classes.secondary
-          : classes.tertiary
-      }
-      {...rest}
-    >
-      {children}
+  const color =
+    buttonType === "primary"
+      ? classes.primary
+      : buttonType === "secondary"
+      ? classes.secondary
+      : classes.tertiary;
+
+  return linkTo ? (
+    <Link href={linkTo}>
+      <MantineButton className={color} {...rest}>
+        {name}
```

Comment:
I though we might want to put icon next to texts of buttons and that's why we put children instead of passing text as a prop. If we don't have the plan, either way is fine tho.

Reply:
You are right. I was only considering text and didn't consider icons. It seems we can also include an icon using props like this:

`<Button icon={<Icon />} />`

Either way is fine for me as well anyway. I just wanted to avoid having to add the <Link> tag every time.

Reply:
I actually found Mantine Button has `leftIcon` and `rightIcon` props. But I like your solution!
https://mantine.dev/core/button/?t=props

Reply:
Oh I didn't know they have props for icons! Let's use their prop then 👍